src/cogent3/core/new_alignment.py

```diff
@@ -2732,25 +2865,31 @@ def get_gapped_seq_bytes(
 
     def add_seqs(
         self, seqs: dict[str, StrORArray], force_unique_keys=True
-    ) -> AlignedData:
-        """Returns a new AlignedData object with added sequences."""
+    ) -> AlignedSeqsData:
+        """Returns a new AlignedSeqsData object with added sequences."""
```

Comment:
assert that the new aligned sequences have the same `align_len` as self. Let the user concatenate things if they want to.